CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+**Thanks for taking the time to contribute! ❤️**
+
+All types of contributions are encouraged and valued, no matter if it's a bug
+report 🐛, a feature request 💡, or a Pull Request 🚀.
+
+- **❓ I have a question:** [Open an Issue]
+- **🐛 I found a bug:** [Open an Issue]
+- **💡 I have an idea:** [Open an Issue]
+- **💻 I want to code:** See below
+
+If you like the project, but just don't have time to contribute, that's OK too!
+You can also star the project ⭐, rave about it online 💬, or add a link to us
+🔗 in your project's readme.
+
+⚠️ You must never report security 🔒 related issues, vulnerabilities or bugs
+including sensitive information to the issue tracker, or elsewhere in public.
+Instead, please [report them to the Microsoft Security Response Center]. You can
+read more about our security policy in [`SECURITY.md`].
+
+## Contributing code
+
+👨‍💼 This repository is a collection of devcontainer features managed by
+devcontainer spec maintainers. Even though **we do not accept new features in
+this repository**, we welcome contributions to [our existing features]!
+
+🚀 If you're looking to create a new feature, we encourage you to [create your
```

Comment:
Would recommend linking to the feature-starter repo: https://github.com/devcontainers/feature-starter.

Reply:
Is it sufficient to link here?

![image](https://user-images.githubusercontent.com/61068799/225144792-8431d0db-f852-4003-a83b-e247ca9b51a6.png)

Reply:
Fixed in https://github.com/jcbhmr/features/commit/b3f4b8b975087af2029bea3581dcb15747bbbb74